server/views/helpers/helpers.ejs

```diff
@@ -19,27 +19,6 @@
       'TRAINING_INTERNAL': 'training-internal',
     }[type] || 'otherType');
 
-  getRelativeModifiedDate = (moment, modifiedDate) => {
-
-    var s = ( moment().diff(moment(modifiedDate)))/ 1000,
-      m = s / 60,
-      h = m / 60,
-      d = h / 24,
-      y = d / 365.242199,
-      tmp;
-
-    return ( tmp = Math.round(s) ) === 1 ? 'just now'
```

Comment:
switched to using `fromNow` instead rather than calculating like this

Reply:
👍

Reply:
yes, good point.  Will fix in future PR